src/navigation/components/Routes.js

```diff
@@ -107,6 +108,17 @@ const paths = [
     showBreadcrumbs: true,
     breadcrumbsLabel: 'group.breadcrumbs_visualization',
   }),
+  path(
+    '/groups/:groupSlug/shared-resource/download/:shareUuid',
```

Comment:
suggestion: Simplify the path. (Same for other instances.)
```suggestion
    '/groups/:groupSlug/download/:shareUuid',
```